docs/extensibility/README.md

```diff
@@ -0,0 +1,15 @@
+# Extensibility
+
+## Overview
+
+Extensibility in Busola allows you to create a UI page for your Custom Resource Defintion (CRD). You can create it at a cluster or Namespace level. To configure the UI page of your CRD, you can add navigation nodes, lists, and details pages, as well as create and edit forms. You can also add [display](display-widgets.md) and [form](form-widgets.md) widgets to have a graphical representation of the elements.
+
+## Getting started
+
+To automatically add the UI page for your CRD, follow these steps:
+
+1.  In the **Cluster Details** view, go to **Configuration > Custom Resource Defintions** and click on your CRD.
+2.  Click **Create UI +**, and provide **Name** and **Category**.
+3.  Optionally, you can uncheck the UI elements that are not necessary.
+
+Alternatively, you can create from scratch a Config Map in the `kube-public` Namespace with a label `busola.io/extension=resource`, and add the required fields to define how to handle your CRD. For more information, visit [Config Map for resource-based extensions](resources.md).
```

Comment:
this doesn't sound good to me... "create a Config Map from scratch" instead?

Also, this still feels like the extra docs are only relevant if you want to create the config map from scratch - when more often than not they'll still be needed to tweak the config map even if auto-generated.
The "Create UI" interface only gives very basic suggestions and quite possibly won't fully match the needs. It's function is more to create a skeleton to improve upon rather than final solution.